src/libstd/sync/mpsc/mod.rs

```diff
@@ -306,6 +306,14 @@ pub struct Iter<'a, T: 'a> {
     rx: &'a Receiver<T>
 }
 
+/// An owning iterator over messages on a receiver, this iterator will block
+/// whenever `next` is called, waiting for a new message, and `None` will be
+/// returned when the corresponding channel has hung up.
+#[stable(feature = "rust1_1", since = "1.1.0")]
```

Comment:
Ah sorry by new feature name I meant more something along the lines of `receiver_into_iter` or something like that